tsconfig.build.json

```
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
```

Biome (2.1.2)

[error] The key compilerOptions was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)